apps/reactotron-app/src/renderer/contexts/Standalone/useStandalone.ts

```diff
@@ -97,7 +97,7 @@ export function reducer(state: State, action: Action) {
           )
         }
 
-        // TODO: Figure out if we can stop having these dumb commands so early. Make core client only send once we actually have a client ID! (maybe won't work for flipper if it doesn't assign client id? Maybe that is how this broke?!?)
+        // TODO: Figure out if we can stop having these dumb commands so early. Make core client only send once we actually have a client ID!
```

Comment:
I did not investigate why we have these commands so early. I only dropped the reference to flipper.